pyqt-apps/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pydm>=1.8.0
 QtAwesome>=0.7.2
+pyqtgraph>=0.11.0
```

Comment:
@anacso17, did you update this version in lnls-ansible too?

Reply:
Yes, @fernandohds564 ! In this PR: https://github.com/lnls-sirius/lnls-ansible/pull/134